style.css

```diff
@@ -213,6 +213,78 @@ nav li a {
   background-color: #008552;
 }
 
+.works-window {
+  position: fixed;
+  top: 1em;
+  padding: 1em;
+  border-radius: 8px;
+  height: 95%;
+  background-color: #fff;
+}
+
+.works-window-elements {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  height: 100%;
+  overflow: scroll;
+  gap: 0.7em;
+}
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 

- Please respect the font size of your text when we are on Mobile and Desktop size. 🤔 
- It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/231304823-b005aae2-c9e3-4307-ba9b-606cb3bdcb03.png) | ![181](https://user-images.githubusercontent.com/79058364/189106086-ad46721f-211a-41e9-97b7-421fb32231c2.png) 


- [ ] Be sure to update your desktop design. The button must be positioned at the top right when you are on the size of the desk


Your Implementation | Figma Design
:---:|:---:
![11](https://user-images.githubusercontent.com/79058364/231305864-c196a7c1-e422-416a-ab73-5e5bc9ed3b44.png)| ![18](https://user-images.githubusercontent.com/79058364/189106145-f947cebe-3a5a-45a8-9851-ada6c6544c46.png)